src/components/ApplicationFooter.jsx

```diff
@@ -3,17 +3,127 @@ import {
   Box,
   Grid,
   Typography,
-  useMediaQuery,
-  useTheme,
+  CardMedia,
+  makeStyles,
 } from "@material-ui/core";
 import { Link } from "react-router-dom";
 import LogoCHWL from "../assets/LogoCHWL.png";
 import { useSelector } from "react-redux";
 
+const useStyles = makeStyles((theme) => ({
+  footerContainer: {
+    [theme.breakpoints.up("xs")]: {
+      display: "flex",
+      flexDirection: "column",
+      width: "90%",
+      maxWidth: "800px",
+      marginLeft: "auto",
+      marginRight: "auto",
+      position: "relative",
+      bottom: "0",
```

Comment:
why don't have it on larger screens?

Reply:
when its on xs and nothing else it gets to everything thats above xs as well